subsys/bluetooth/services/ias/ias_internal.h

```diff
@@ -0,0 +1,17 @@
+/* @file
+ * @brief Immediate Alert Service  internal header file
+ *
+ * For use with the Immediate Alert Service (IAS)
+ *
+ * Copyright (c) 2022 Codecoup
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/** @brief Immediate Alert Service alert levels */
+enum bt_ias_alert_lvl {
+	BT_IAS_ALERT_LVL_NO_ALERT,
+	BT_IAS_ALERT_LVL_MILD_ALERT,
+	BT_IAS_ALERT_LVL_HIGH_ALERT,
+} __packed;
```

Comment:
Wait, I'm confused, I thought that these were not useful for the public API? 

But it seems that they are being used by `bt_ias_client_alert_write`, right? 

So they should moved to the public header file, and `bt_ias_client_alert_write` should take `enum bt_ias_alert_lvl` as input value, and not `uint8_t`. 

It also does not need to be packed.

Reply:
Well, I was confused about it also - but the way that `bt_ias_client_alert_write` was written was meant to take this enum so this may be clearer and somewhat more elegant to someone looking at this code. Also without this public enum with alert values, I had to define alert levels in the shell. I'll roll back this change and move alert levels as an enum into the public header, we can always discuss it later, perhaps during the weekly meeting.